.changeset/tiny-pianos-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+
+---
+
+fix(deps): update dependency @commercetools-frontend/babel-preset-mc-app to v20
```

Comment:
The text linter is failing here with

```
Use 'Merchant Center' instead   commercetools.Product 
              of 'mc'. 
```

I am tempted to ignore files in `.changeset` folder. What do you think @nkuehn  ?

Reply:
Make the technical package name a string literal in MD, it's correct (not natural language) and will exclude it from the linting. 

```markdown
will be linted as natural language:  "@commercetools-frontend/babel-preset-mc-app"

will not be linted as natural language:  `@commercetools-frontend/babel-preset-mc-app`
```